ntbs-service/Pages/Notifications/Edit/Patient.cshtml.cs

```diff
@@ -12,7 +12,7 @@
 
 namespace ntbs_service.Pages.Notifications.Edit
 {
-    public class PatientModel : NotificationEditModelBase
+    public class PatientDetailsModel : NotificationEditModelBase
```

Comment:
Is the view model name change just a by-product of a replace all? Or is it intentional?

Reply:
It's an intentional change just to sync up using PatientDetails instead of Patient to match the database model, possibly worth changing file name too?

Reply:
I think if we're changing the view model name, and definitely if we're changing the filename - we should adjust the routing to match too?

Reply:
This is very true - @jasiekmiko do you have any strong feelings on renaming view model name, file name and routing to patientdetails or leaving it all as patient?

Reply:
yeah, let's get them synced us since we're half way there. syncing routing seems right!